Source/Core/Core/HW/SI/SI_DeviceGBA.cpp

```diff
@@ -329,7 +343,23 @@ int CSIDevice_GBA::RunBuffer(u8* buffer, int request_length)
 
   case NextAction::ReceiveResponse:
   {
-    int num_data_received = m_sock_server.Receive(buffer);
+    u8 bytes = 1;
+    switch (m_last_cmd)
+    {
+    case CMD_RESET:
+    case CMD_STATUS:
+      bytes = 3;
```

Comment:
Is it worth defining named constants for these?

Reply:
I'm not really sure. It''d just end up being e.g. CMD_RESET_BYTES_OUT, CMD_RESET_BYTES_IN, CMD_STATUS_BYTES_OUT, etc etc. Some could be reused with the transfer interval code but not all.